src/config.d.ts

```diff
@@ -10,6 +10,7 @@ interface Config {
   REACT_APP_OS_WEB_API_URL: string;
   REACT_APP_SEARCH_URL: string;
   REACT_APP_HIGHLIGHTS_URL: string;
+  ORIGIN_URL: string;
```

Comment:
origin url for what? is this used for loading the rex configs? oh i see below you're handling undefined window, is this only for loading rex configs when pre-rendering? because when not pre-rendering you could just not put a host and use a relative url like `/rex/release.json`

when pre-rendering our local config will definitely be newer than the remote, we should not go out of our way to make fetching the remote work. in the gateway we could skip the remote check if window is undefined

Reply:
if you're not using the gateway in the scripts, then you don't need the skip check in the gateway, but i don't understand why you need the origin url config, you can just reference with a relative url like `/rex/release.json`

Reply:
I think I've suggested adding it to the config since I saw a bunch of this type of variables there, but indeed relative url seems better